pkg/domain/entities/engine_container.go

```diff
@@ -28,6 +28,7 @@ type ContainerEngine interface {
 	ContainerLogs(ctx context.Context, containers []string, options ContainerLogsOptions) error
 	ContainerMount(ctx context.Context, nameOrIds []string, options ContainerMountOptions) ([]*ContainerMountReport, error)
 	ContainerPause(ctx context.Context, namesOrIds []string, options PauseUnPauseOptions) ([]*PauseUnpauseReport, error)
+	ContainerPort(ctx context.Context, options ContainerPortOptions) ([]*ContainerPortReport, error)
```

Comment:
```suggestion
	ContainerPort(ctx context.Context, nameOrId string, options ContainerPortOptions) ([]*ContainerPortReport, error)
```
Would be more consistent with other methods.